modules/nw-scaling-ingress-controller.adoc

```diff
@@ -0,0 +1,43 @@
+// Module filename: nw-scaling-ingress-controller.adoc
+// Module included in the following assemblies:
+// * networking/ingress-controller-configuration.adoc
+
+[id="nw-ingress-controller-configuration-{context}"]
+= Scaling an ingress controller
+
+An ingress controller can be manually scaled up or down based on
+requirements such as the need to increase throughput. `oc`
+commands are used to scale the `IngressController` resource.
+The following procedure provides an example for scaling up the
+default `IngressController`.
+
+.Procedure
+. View the current number of available replicas for the default `IngressController`.
++
+----
+$ oc get -n openshift-ingress-operator ingresscontrollers/default \
+-o jsonpath='{$.status.availableReplicas}'
```

Comment:
Not suggesting a change here — just for my own reference going forward to be consistent — what's a good wrapping convention? e.g. wrap at 80 characters?

Reply:
IMO we should wrap at 80 characters. With that said, code blocks seem to vary from one project to the next.

@bmcelvee I do not see a standard approach to line wrapping taken by other docs in the repo. Is there a docs standard we should follow?